src/styles/calendar.scss

```
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+
+  width: ($cell-size * 7) + $gap-shift;
+
+  &__day {
+    @include square($cell-size);
+    box-sizing: border-box;
+    border: 1px solid black;
+    font-size: 30px;
+    line-height: 100px;
+    background-color: #eee;
+    text-align: center;
+    cursor: pointer;
+    transition-duration: 0.5s;
```

```suggestion
    transition-duration: transform 0.5s, background-color 0.5s;
```
And move 0.5s to the variable